internal/tablemap_test.go

```diff
@@ -91,3 +91,32 @@ func TestGetSpannerCol(t *testing.T) {
 		assert.Equal(t, spCol, spCol2, tc.name)
 	}
 }
+
+func TestGetSpannerKeyName(t *testing.T) {
+	schemaForeignKeys := make(map[string]bool)
+
+	basicTests := []struct {
+		name       string // Name of test.
+		srcKeyName string // Source foreign key name.
+		error      bool   // Whether an error is expected.
```

Comment:
The error fields is no longer needed since GetSpannerKeyName doesn't return an error.

Reply:
yes, removed error field